### DISCUSSION TITLE
Pinning happens per monorepo

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#11438] [Request Help] [answered]
I noticed that if a project uses monorepos, Renovate will create separate "Pin dependencies" PRs for the different monorepos. With multiple monorepos in use (like angular, angular-cli and ionic-native for an npm project), this means that every time you accept one PR, all the others are conflicted and need to be recreated. With a PR hourly limit set, it takes quite some time before Renovate even considers actually updating dependencies.

I'd expect that the pinning can be done for all dependencies in one go, I don't see why it needs to be separate. Can someone explain why it's split up, and whether there's anything I can configure to join all dependency pinning in one PR?

**Comment by rarkins** [comment, marked as answer]
It's not expected -have you customised branch prefix or additional prefix?

**Reply by astellingwerf** [reply]
No, I have not. The project has no configuration at all, and the only global configuration I can imagine as remotely relevant is the following:
```
    'extends': [
        'config:base',
        ':separateMultipleMajorReleases',
        ':prHourlyLimit4'
    ],```

**Reply by rarkins** [reply]
What are the branch names of the PRs which you have expected to be combined?

**Reply by astellingwerf** [reply]
They are 'renovate/all', 'renovate/angular-cli-monorepo', and 'renovate/angular-monorepo'.

**Reply by rarkins** [reply]
Even though they contain pinning?

**Reply by astellingwerf** [reply]
Yes!
<img width="1135" alt="Schermafbeelding 2021-08-28 om 16 36 39" src="https://user-images.githubusercontent.com/1637358/131221556-8b886009-4ce8-4dfe-87bc-f62fb4e4c0f0.png">

If that helps: I'm running against BitBucket cloud and with Renovate 26.7.0.

**Reply by rarkins** [reply]
Can you file this as a bug report issue, assuming you can create a minimal reproduction?

**Reply by astellingwerf** [reply]
Will do, thanks @rarkins. Need to work on the latter half first, but if I can do that, I will share it.

**Reply by astellingwerf** [reply]
I reported #11680.
